apps/trading/e2e/tests/fees/test_fees.py

```diff
@@ -645,6 +656,7 @@ def test_fills_maker_fee_tooltip_discount_program(
     )
 
 
+@pytest.mark.xdist_group(name="test_fees")
```

Comment:
Nice, presumably this groups for parallelization?

Reply:
Yeah, if you use groups to specifiy parralelisation, anything that isn't given a group will just be run randomly (not grouped by file etc). So this is required to make sure tests under the same group name are run in the same thread